TestCases/py_wrapper/updated_moving_frame_NACA12/config.cfg

```diff
@@ -0,0 +1,75 @@
+% PROBLEM DEFINITION
+%
+SOLVER= EULER
```

Comment:
This is the same config right? If so, please use a relative path to use it directly without making a copy

Reply:
I modified the onflow and shifted the origin to obtain results which are more clear to interpret.